README.md

```diff
@@ -217,6 +218,27 @@ export default connect(
 )(Component);
 ```
 
+### Binding actions to a outside object
```

Comment:
I would change this to: 

`Binding actions outside your application scope`, what you think?